pattern-library/js/select-replace.js

```diff
@@ -60,16 +57,21 @@ define([
 
                     $el.replaceWith([
                         '<div class="' + variables.wrapperClass + '">',
-                            '<select class="' + replaced[0].className + ' is-replaced" id="' + replaced[0].id + '" name="' + replaced[0].name + '" ' + disabled + '>' + replaced[0].innerHTML + '</select>',
-                            '<span class="' + variables.customClass + ' ' + statuses.join(' ') + '" aria-hidden="true">',
-                                '<span class="' + variables.valueClass + '">' + CustomSelectReplacement.setInitialText($el) + '</span>',
-                                '<span class="icon-fallback-glyph">',
-                                    '<span class="icon ' + variables.iconClass + '" aria-hidden="true"></span>',
-                                    '<span class="text">Down arrow</span>',
-                                '</span>',
-                            '</span>',
+                        '<select class="' + replaced[0].className + ' is-replaced" id="' + replaced[0].id +
+                        '" name="' + replaced[0].name + '" ' + disabled + '>' +
+                        replaced[0].innerHTML +
+                        '</select>',
+                        '<span class="' + variables.customClass + ' ' + statuses.join(' ') + '" aria-hidden="true">',
+                        '<span class="' + variables.valueClass + '">' +
+                            CustomSelectReplacement.setInitialText($el) +
+                        '</span>',
+                        '<span class="icon-fallback-glyph">',
+                        '<span class="icon ' + variables.iconClass + '" aria-hidden="true"></span>',
+                        '<span class="text">Down arrow</span>',
+                        '</span>',
+                        '</span>',
```

Comment:
How about:

```javascript
/* eslint-disable indent */
$el.replaceWith([
    '<div class="' + variables.wrapperClass + '">',
        '<select class="' + replaced[0].className + ' is-replaced" id="' + replaced[0].id + '" name="' + replaced[0].name + '" ' + disabled + '>' + replaced[0].innerHTML + '</select>',
        '<span class="' + variables.customClass + ' ' + statuses.join(' ') + '" aria-hidden="true">',
            '<span class="' + variables.valueClass + '">' + CustomSelectReplacement.setInitialText($el) + '</span>',
            '<span class="icon-fallback-glyph">',
                '<span class="icon ' + variables.iconClass + '" aria-hidden="true"></span>',
                '<span class="text">Down arrow</span>',
            '</span>',
        '</span>',
    '</div>'
].join(''));
/* eslint-enable indent */
```

This is a case I think where the indentation, though technically incorrect, is worth keeping.